AK/Random.h

```diff
@@ -47,7 +47,7 @@ inline void fill_with_random(void* buffer, size_t length)
 {
 #if defined(__serenity__)
     arc4random_buf(buffer, length);
-#elif defined(__linux__) or defined(__APPLE__)
+#elif defined(__unix__) or defined(__APPLE__)
     int rc = getentropy(buffer, length);
```

Comment:
Actually I think NetBSD might not have `getentropy()` but would define `__unix__`

Reply:
Not that I've ever heard from anyone building on NetBSD. 🤷🏻‍♂️

Reply:
Could we assume that getrandom would work fine on NetBSD, or would you rather either throw a preprocessor error if __NetBSD__ is defined, or leave it as is and let any eventual NetBSD user deal with the non-existent getentropy function?